tempto-core/build.gradle

```diff
@@ -74,3 +108,56 @@ license {
 }
 
 build.dependsOn.add(buildFatJar)
+
+// publishing to sonatype
+
+nexusStaging {
+  packageGroup = group
+  stagingProfileId = '176117bc76f90b'
+}
+
+signing {
+  sign configurations.archives
+}
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+      repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+        authentication(userName: nexusUsername, password: nexusPassword)
+      }
+
+      snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
+        authentication(userName: nexusUsername, password: nexusPassword)
+      }
+
+      pom.project {
+        name 'tempto-core'
+        packaging 'jar'
+        artifactId 'tempto-core'
+        description 'Tempto - test framework'
+        url 'https://github.com/prestodb/tempto'
+        scm {
+          url 'git@github.com:prestodb/tempto.git'
+          connection 'scm:git:git@github.com:prestodb/tempto.git'
+          developerConnection 'scm:git:git@github.com:prestodb/tempto.git'
+        }
+        licenses {
+          license {
+            name 'The Apache Software License, Version 2.0'
+            url 'http://www.apache.org/license/LICENSE-2.0.txt'
+          }
+        }
+        developers {
+          developer {
+            id 'bdch'
```

Comment:
I would rather avoid TDCH which stands for Teradata Connector for Hadoop (the Ryan's project). TCH? :)